Strategy 'BodyOnlyIfStatusAtLeastStrategy' does not log request attributes

<!--- Provide a general summary of the issue in the Title above -->

## Description
In our project we need to log the attributes which extracted from JWT token. When using BodyOnlyIfStatusAtLeastStrategy, the request attributes does not be logged when the response status is less than the minimum status.



## Expected Behavior
withoutBody() should remove only the body from the request but not the attributes.

## Actual Behavior
log without body removed body and attributes

## Possible Fix

We found in this class, the request.withoutBodhy() removed the attributes from the request object.
https://github.com/zalando/logbook/blob/main/logbook-core/src/main/java/org/zalando/logbook/core/BodyOnlyIfStatusAtLeastStrategy.java#L34C27-L35C10

## Steps to Reproduce

1. log with strategy BodyOnlyIfStatusAtLeastStrategy and status=400
2. exract attributes with conf  

```
logbook:
  strategy: body-only-if-status-at-least
  minimum-status: 400
  attribute-extractors:
    - type: JwtAllMatchingClaimsExtractor
      claim-names: [ "xxx" ]

```

3. log the information

Thanks ^^

Ok thanks, lets see the context as you requested:
dependencies logbook-spring-boot-starter & logbook-spring-boot-webflux-autoconfigure are used
to log the request and response of the API calls.

logbook configuration in the application.yaml
```
logbook:
  strategy: body-only-if-status-at-least
  minimum-status: 400
  predicate:
    exclude:
      - path: /restricted/actuator/**
  # Extraire le sub & work_company du JWT token pour les logger
  attribute-extractors:
    - type: JwtAllMatchingClaimsExtractor
      claim-names: [ "sub", "work_company" ]
```

Request and response log output when the status is 200 ( < 400)
**The request attributes are not logged**
```
{"@timestamp":"2025-02-11T18:07:41.8066314+01:00","@version":"1","message":"GET http://localhost:8080/v3/journeys?trainNumber=7611&startDate=2025-03-03","logger_name":"org.zalando.logbook.Logbook","thread_name":"reactor-http-nio-9","level":"TRACE","level_value":5000,"http":{"origin":"remote","type":"request","correlation":"a8307a5180988d13","protocol":"HTTP/1.1","remote":"/[0:0:0:0:0:0:0:1]:64803","method":"GET","uri":"http://localhost:8080/v3/journeys?trainNumber=7611&startDate=2025-03-03","host":"localhost","path":"/v3/journeys","scheme":"http","port":"8080","headers":{"Accept":["application/json, text/plain, */*"],"Accept-Encoding":["gzip, compress, deflate, br"],"Authorization":["XXX"],"Connection":["keep-alive"],"Host":["localhost:8080"],"request-start-time":["1739293661051"],"User-Agent":["bruno-runtime/1.38.1"]}}}
{"@timestamp":"2025-02-11T18:07:41.8066314+01:00","@version":"1","message":"200 OK GET http://localhost:8080/v3/journeys?trainNumber=7611&startDate=2025-03-03","logger_name":"org.zalando.logbook.Logbook","thread_name":"reactor-http-nio-9","level":"TRACE","level_value":5000,"http":{"origin":"local","type":"response","correlation":"a8307a5180988d13","duration":749,"protocol":"HTTP/1.1","status":200,"headers":{"Cache-Control":["no-cache, no-store, max-age=0, must-revalidate"],"Content-Length":["806221"],"Content-Security-Policy":["script-src 'self'"],"Content-Type":["application/json"],"ETag":["\"d08c68c8023c689e3ed13c5b815716da\""],"Expires":["0"],"Pragma":["no-cache"],"Referrer-Policy":["no-referrer"],"Vary":["Origin","Access-Control-Request-Method","Access-Control-Request-Headers"],"X-Content-Type-Options":["nosniff"],"X-Frame-Options":["DENY"],"X-XSS-Protection":["1; mode=block"]}}}
```

Request and response log output when the status is 500 ( > 400)
**The request attributes are logged correcty**
```
{"@timestamp":"2025-02-11T18:07:51.6499429+01:00","@version":"1","message":"GET http://localhost:8080/v3/journeys?trainNumber=76111&startDate=2025-03-03","logger_name":"org.zalando.logbook.Logbook","thread_name":"reactor-http-nio-11","level":"TRACE","level_value":5000,"http":{"origin":"remote","type":"request","correlation":"ddef2b5860fb4141","protocol":"HTTP/1.1","remote":"/[0:0:0:0:0:0:0:1]:64842","method":"GET","uri":"http://localhost:8080/v3/journeys?trainNumber=76111&startDate=2025-03-03","host":"localhost","path":"/v3/journeys","scheme":"http","port":"8080","attributes":{"work_company":"[\"SNCF VOYAGEURS\"]","sub":"POGT03221"},"headers":{"Accept":["application/json, text/plain, */*"],"Accept-Encoding":["gzip, compress, deflate, br"],"Authorization":["XXX"],"Connection":["keep-alive"],"Host":["localhost:8080"],"request-start-time":["1739293671562"],"User-Agent":["bruno-runtime/1.38.1"]}}}
{"@timestamp":"2025-02-11T18:07:51.6561893+01:00","@version":"1","message":"500 Internal Server Error GET http://localhost:8080/v3/journeys?trainNumber=76111&startDate=2025-03-03","logger_name":"org.zalando.logbook.Logbook","thread_name":"reactor-http-nio-11","level":"TRACE","level_value":5000,"http":{"origin":"local","type":"response","correlation":"ddef2b5860fb4141","duration":82,"protocol":"HTTP/1.1","status":500,"headers":{"Cache-Control":["no-cache, no-store, max-age=0, must-revalidate"],"Content-Length":["49837"],"Content-Security-Policy":["script-src 'self'"],"Content-Type":["application/json"],"Expires":["0"],"Pragma":["no-cache"],"Referrer-Policy":["no-referrer"],"Vary":["Origin","Access-Control-Request-Method","Access-Control-Request-Headers"],"X-Content-Type-Options":["nosniff"],"X-Frame-Options":["DENY"],"X-XSS-Protection":["1; mode=block"]},"body":{"timestamp":"2025-02-11T17:07:51.648918Z","path":"/v3/journeys","status":500,"code":"INTERNAL_SERVER_ERROR","message":"<html>\n<head>\n<meta http-equiv=\"Content-Type\" content=\"text/html;charset=ISO-8859-1\"/>\n<title>Error 500 java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</title>\n</head>\n<body><h2>HTTP ERROR 500 java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</h2>\n<table>\n<tr><th>URI:</th><td>/gorail</td></tr>\n<tr><th>STATUS:</th><td>500</td></tr>\n<tr><th>MESSAGE:</th><td>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</td></tr>\n<tr><th>SERVLET:</th><td>com.github.tomakehurst.wiremock.servlet.WireMockHandlerDispatchingServlet-4201a617</td></tr>\n<tr><th>CAUSED BY:</th><td>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</td></tr>\n</table>\n<h3>Caused by:</h3><pre>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)\n\tat java.base/java.io.FileInputStream.open0(Native Method)\n\tat java.base/java.io.FileInputStream.open(Unknown Source)\n\tat java.base/java.io.FileInputStream.&lt;init&gt;(Unknown Source)\n\tat java.base/java.io.FileInputStream.&lt;init&gt;(Unknown Source)\n\tat java.base/sun.net.www.protocol.file.FileURLConnection.connect(Unknown Source)\n\tat java.base/sun.net.www.protocol.file.FileURLConnection.getInputStream(Unknown Source)\n\tat java.base/java.net.URL.openStream(Unknown Source)\n\tat com.github.tomakehurst.wiremock.common.BinaryFile.getStream(BinaryFile.java:56)\n\tat com.github.tomakehurst.wiremock.common.BinaryFile.readContents(BinaryFile.java:33)\n\tat com.github.tomakehurst.wiremock.common.TextFile.readContentsAsString(TextFile.java:30)\n\tat com.github.tomakehurst.wiremock.extension.responsetemplating.ResponseTemplateTransformer.transform(ResponseTemplateTransformer.java:101)\n\tat com.github.tomakehurst.wiremock.stubbing.AbstractStubMappings.applyV2Transformations(AbstractStubMappings.java:152)\n\tat com.github.tomakehurst.wiremock.stubbing.AbstractStubMappings.serveFor(AbstractStubMappings.java:110)\n\tat com.github.tomakehurst.wiremock.core.WireMockApp.serveStubFor(WireMockApp.java:293)\n\tat com.github.tomakehurst.wiremock.http.StubRequestHandler.handleRequest(StubRequestHandler.java:73)\n\tat com.github.tomakehurst.wiremock.http.AbstractRequestHandler.handle(AbstractRequestHandler.java:73)\n\tat com.github.tomakehurst.wiremock.servlet.WireMockHandlerDispatchingServlet.service(WireMockHandlerDispatchingServlet.java:165)\n\tat wiremock.jakarta.servlet.http.HttpServlet.service(HttpServlet.java:587)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:764)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1665)\n\tat wiremock.org.eclipse.jetty.servlets.CrossOriginFilter.handle(CrossOriginFilter.java:314)\n\tat wiremock.org.eclipse.jetty.servlets.CrossOriginFilter.doFilter(CrossOriginFilter.java:267)\n\tat wiremock.org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:202)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1635)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:527)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:221)\n\tat wiremock.org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1381)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:176)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:484)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:174)\n\tat wiremock.org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1303)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:129)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)\n\tat wiremock.org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:822)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:141)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)\n\tat wiremock.org.eclipse.jetty.server.Server.handle(Server.java:563)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel$RequestDispatchable.dispatch(HttpChannel.java:1598)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:753)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:501)\n\tat wiremock.org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:287)\n\tat wiremock.org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314)\n\tat wiremock.org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100)\n\tat wiremock.org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.runTask(AdaptiveExecutionStrategy.java:421)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.consumeTask(AdaptiveExecutionStrategy.java:390)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.tryProduce(AdaptiveExecutionStrategy.java:277)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.run(AdaptiveExecutionStrategy.java:199)\n\tat wiremock.org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:411)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:969)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1194)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1149)\n\tat java.base/java.lang.Thread.run(Unknown Source)\n</pre>\n\n</body>\n</html>\n","details":"org.springframework.web.reactive.function.client.WebClientResponseException$InternalServerError: 500 Internal Server Error from POST https://develop-sirius-mocks.sncf-dsi-pr-dev.aws.vsct.fr/gorail","trace":"sncf.traction.sirius.journey.gorail.JourneyCommonError$InternalServerError: <html>\n<head>\n<meta http-equiv=\"Content-Type\" content=\"text/html;charset=ISO-8859-1\"/>\n<title>Error 500 java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</title>\n</head>\n<body><h2>HTTP ERROR 500 java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</h2>\n<table>\n<tr><th>URI:</th><td>/gorail</td></tr>\n<tr><th>STATUS:</th><td>500</td></tr>\n<tr><th>MESSAGE:</th><td>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</td></tr>\n<tr><th>SERVLET:</th><td>com.github.tomakehurst.wiremock.servlet.WireMockHandlerDispatchingServlet-4201a617</td></tr>\n<tr><th>CAUSED BY:</th><td>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</td></tr>\n</table>\n<h3>Caused by:</h3><pre>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)\n\tat java.base/java.io.FileInputStream.open0(Native Method)\n\tat java.base/java.io.FileInputStream.open(Unknown Source)\n\tat java.base/java.io.FileInputStream.&lt;init&gt;(Unknown Source)\n\tat java.base/java.io.FileInputStream.&lt;init&gt;(Unknown Source)\n\tat java.base/sun.net.www.protocol.file.FileURLConnection.connect(Unknown Source)\n\tat java.base/sun.net.www.protocol.file.FileURLConnection.getInputStream(Unknown Source)\n\tat java.base/java.net.URL.openStream(Unknown Source)\n\tat com.github.tomakehurst.wiremock.common.BinaryFile.getStream(BinaryFile.java:56)\n\tat com.github.tomakehurst.wiremock.common.BinaryFile.readContents(BinaryFile.java:33)\n\tat com.github.tomakehurst.wiremock.common.TextFile.readContentsAsString(TextFile.java:30)\n\tat com.github.tomakehurst.wiremock.extension.responsetemplating.ResponseTemplateTransformer.transform(ResponseTemplateTransformer.java:101)\n\tat com.github.tomakehurst.wiremock.stubbing.AbstractStubMappings.applyV2Transformations(AbstractStubMappings.java:152)\n\tat com.github.tomakehurst.wiremock.stubbing.AbstractStubMappings.serveFor(AbstractStubMappings.java:110)\n\tat com.github.tomakehurst.wiremock.core.WireMockApp.serveStubFor(WireMockApp.java:293)\n\tat com.github.tomakehurst.wiremock.http.StubRequestHandler.handleRequest(StubRequestHandler.java:73)\n\tat com.github.tomakehurst.wiremock.http.AbstractRequestHandler.handle(AbstractRequestHandler.java:73)\n\tat com.github.tomakehurst.wiremock.servlet.WireMockHandlerDispatchingServlet.service(WireMockHandlerDispatchingServlet.java:165)\n\tat wiremock.jakarta.servlet.http.HttpServlet.service(HttpServlet.java:587)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:764)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1665)\n\tat wiremock.org.eclipse.jetty.servlets.CrossOriginFilter.handle(CrossOriginFilter.java:314)\n\tat wiremock.org.eclipse.jetty.servlets.CrossOriginFilter.doFilter(CrossOriginFilter.java:267)\n\tat wiremock.org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:202)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1635)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:527)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:221)\n\tat wiremock.org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1381)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:176)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:484)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:174)\n\tat wiremock.org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1303)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:129)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)\n\tat wiremock.org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:822)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:141)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)\n\tat wiremock.org.eclipse.jetty.server.Server.handle(Server.java:563)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel$RequestDispatchable.dispatch(HttpChannel.java:1598)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:753)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:501)\n\tat wiremock.org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:287)\n\tat wiremock.org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314)\n\tat wiremock.org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100)\n\tat wiremock.org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.runTask(AdaptiveExecutionStrategy.java:421)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.consumeTask(AdaptiveExecutionStrategy.java:390)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.tryProduce(AdaptiveExecutionStrategy.java:277)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.run(AdaptiveExecutionStrategy.java:199)\n\tat wiremock.org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:411)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:969)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1194)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1149)\n\tat java.base/java.lang.Thread.run(Unknown Source)\n</pre>\n\n</body>\n</html>\n\r\n\tat sncf.traction.sirius.journey.gorail.GorailJourneyService$searchSillonsDasSferaWithStatus-fSDxh-Q$$inlined$coroutineBinding$1.invokeSuspend(CoroutineBinding.kt:112)\r\n\tSuppressed: The stacktrace has been enhanced by Reactor, refer to additional information below: \r\nError has been observed at the following site(s):\r\n\t*__checkpoint ⇢ Handler sncf.traction.sirius.journey.JourneyController#getJourneyWithStatus(String, LocalDate, String, String, String, String, ServerWebExchange, Principal, Continuation) [DispatcherHandler]\r\nOriginal Stack Trace:\r\n\t\tat sncf.traction.sirius.journey.gorail.GorailJourneyService$searchSillonsDasSferaWithStatus-fSDxh-Q$$inlined$coroutineBinding$1.invokeSuspend(CoroutineBinding.kt:112)\r\n\t\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\r\n\t\tat kotlinx.coroutines.DispatchedTaskKt.resume(DispatchedTask.kt:233)\r\n\t\tat kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:175)\r\n\t\tat kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:147)\r\n\t\tat kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)\r\n\t\tat kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)\r\n\t\tat kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)\r\n\t\tat kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)\r\n\t\tat kotlinx.coroutines.reactor.MonoKt$awaitSingleOrNull$2$1.onError(Mono.kt:62)\r\n\t\tat reactor.core.publisher.StrictSubscriber.onError(StrictSubscriber.java:106)\r\n\t\tat reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onError(FluxContextWrite.java:121)\r\n\t\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241)\r\n\t\tat reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315)\r\n\t\tat reactor.core.publisher.Operators$MultiSubscriptionSubscriber.onError(Operators.java:2236)\r\n\t\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180)\r\n\t\tat reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onError(MonoIgnoreThen.java:280)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:232)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onComplete(MonoIgnoreThen.java:204)\r\n\t\tat reactor.core.publisher.FluxOnErrorReturn$ReturnSubscriber.onComplete(FluxOnErrorReturn.java:169)\r\n\t\tat reactor.core.publisher.MonoIgnoreElements$IgnoreElementsSubscriber.onComplete(MonoIgnoreElements.java:89)\r\n\t\tat reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onComplete(FluxMapFuseable.java:152)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.core.publisher.Operators.complete(Operators.java:137)\r\n\t\tat reactor.netty.channel.FluxReceive.startReceiver(FluxReceive.java:179)\r\n\t\tat reactor.netty.channel.FluxReceive.subscribe(FluxReceive.java:145)\r\n\t\tat reactor.core.publisher.InternalFluxOperator.subscribe(InternalFluxOperator.java:68)\r\n\t\tat reactor.netty.ByteBufFlux.subscribe(ByteBufFlux.java:340)\r\n\t\tat reactor.core.publisher.Mono.subscribe(Mono.java:4576)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:265)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen.subscribe(MonoIgnoreThen.java:51)\r\n\t\tat reactor.core.publisher.Mono.subscribe(Mono.java:4576)\r\n\t\tat reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)\r\n\t\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241)\r\n\t\tat reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onError(MonoIgnoreThen.java:280)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:232)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onComplete(MonoIgnoreThen.java:204)\r\n\t\tat reactor.core.publisher.FluxOnErrorReturn$ReturnSubscriber.onComplete(FluxOnErrorReturn.java:169)\r\n\t\tat reactor.core.publisher.MonoIgnoreElements$IgnoreElementsSubscriber.onComplete(MonoIgnoreElements.java:89)\r\n\t\tat reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onComplete(FluxMapFuseable.java:152)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.core.publisher.Operators.complete(Operators.java:137)\r\n\t\tat reactor.netty.channel.FluxReceive.startReceiver(FluxReceive.java:179)\r\n\t\tat reactor.netty.channel.FluxReceive.subscribe(FluxReceive.java:145)\r\n\t\tat reactor.core.publisher.InternalFluxOperator.subscribe(InternalFluxOperator.java:68)\r\n\t\tat reactor.netty.ByteBufFlux.subscribe(ByteBufFlux.java:340)\r\n\t\tat reactor.core.publisher.Mono.subscribe(Mono.java:4576)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:265)\r\n\t\tat reactor.core.publisher.MonoIgnoreThen.subscribe(MonoIgnoreThen.java:51)\r\n\t\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:165)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onNext(FluxMap.java:122)\r\n\t\tat reactor.core.publisher.FluxOnErrorReturn$ReturnSubscriber.onNext(FluxOnErrorReturn.java:162)\r\n\t\tat reactor.core.publisher.FluxDefaultIfEmpty$DefaultIfEmptySubscriber.onNext(FluxDefaultIfEmpty.java:122)\r\n\t\tat reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onNext(FluxMapFuseable.java:129)\r\n\t\tat reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onNext(FluxContextWrite.java:107)\r\n\t\tat reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.onNext(FluxMapFuseable.java:299)\r\n\t\tat reactor.core.publisher.FluxFilterFuseable$FilterFuseableConditionalSubscriber.onNext(FluxFilterFuseable.java:337)\r\n\t\tat reactor.core.publisher.Operators$BaseFluxToMonoOperator.completePossiblyEmpty(Operators.java:2097)\r\n\t\tat reactor.core.publisher.MonoCollect$CollectSubscriber.onComplete(MonoCollect.java:145)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.netty.channel.FluxReceive.onInboundComplete(FluxReceive.java:413)\r\n\t\tat reactor.netty.channel.ChannelOperations.onInboundComplete(ChannelOperations.java:455)\r\n\t\tat reactor.netty.channel.ChannelOperations.terminate(ChannelOperations.java:509)\r\n\t\tat reactor.netty.http.client.HttpClientOperations.onInboundNext(HttpClientOperations.java:819)\r\n\t\tat reactor.netty.channel.ChannelOperationsHandler.channelRead(ChannelOperationsHandler.java:115)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat org.zalando.logbook.netty.LogbookClientHandler.channelRead(LogbookClientHandler.java:75)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:289)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)\r\n\t\tat io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1515)\r\n\t\tat io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1378)\r\n\t\tat io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1427)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:530)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:469)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1357)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:868)\r\n\t\tat io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)\r\n\t\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)\r\n\t\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)\r\n\t\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)\r\n\t\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)\r\n\t\tat io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)\r\n\t\tat io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\r\n\t\tat io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)\r\n\t\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\nCaused by: WebClientError(httpStatus=500 INTERNAL_SERVER_ERROR, isIOError=false, body=<html>\n<head>\n<meta http-equiv=\"Content-Type\" content=\"text/html;charset=ISO-8859-1\"/>\n<title>Error 500 java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</title>\n</head>\n<body><h2>HTTP ERROR 500 java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</h2>\n<table>\n<tr><th>URI:</th><td>/gorail</td></tr>\n<tr><th>STATUS:</th><td>500</td></tr>\n<tr><th>MESSAGE:</th><td>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</td></tr>\n<tr><th>SERVLET:</th><td>com.github.tomakehurst.wiremock.servlet.WireMockHandlerDispatchingServlet-4201a617</td></tr>\n<tr><th>CAUSED BY:</th><td>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)</td></tr>\n</table>\n<h3>Caused by:</h3><pre>java.io.FileNotFoundException: /home/wiremock/./__files/gorail/76111.json (No such file or directory)\n\tat java.base/java.io.FileInputStream.open0(Native Method)\n\tat java.base/java.io.FileInputStream.open(Unknown Source)\n\tat java.base/java.io.FileInputStream.&lt;init&gt;(Unknown Source)\n\tat java.base/java.io.FileInputStream.&lt;init&gt;(Unknown Source)\n\tat java.base/sun.net.www.protocol.file.FileURLConnection.connect(Unknown Source)\n\tat java.base/sun.net.www.protocol.file.FileURLConnection.getInputStream(Unknown Source)\n\tat java.base/java.net.URL.openStream(Unknown Source)\n\tat com.github.tomakehurst.wiremock.common.BinaryFile.getStream(BinaryFile.java:56)\n\tat com.github.tomakehurst.wiremock.common.BinaryFile.readContents(BinaryFile.java:33)\n\tat com.github.tomakehurst.wiremock.common.TextFile.readContentsAsString(TextFile.java:30)\n\tat com.github.tomakehurst.wiremock.extension.responsetemplating.ResponseTemplateTransformer.transform(ResponseTemplateTransformer.java:101)\n\tat com.github.tomakehurst.wiremock.stubbing.AbstractStubMappings.applyV2Transformations(AbstractStubMappings.java:152)\n\tat com.github.tomakehurst.wiremock.stubbing.AbstractStubMappings.serveFor(AbstractStubMappings.java:110)\n\tat com.github.tomakehurst.wiremock.core.WireMockApp.serveStubFor(WireMockApp.java:293)\n\tat com.github.tomakehurst.wiremock.http.StubRequestHandler.handleRequest(StubRequestHandler.java:73)\n\tat com.github.tomakehurst.wiremock.http.AbstractRequestHandler.handle(AbstractRequestHandler.java:73)\n\tat com.github.tomakehurst.wiremock.servlet.WireMockHandlerDispatchingServlet.service(WireMockHandlerDispatchingServlet.java:165)\n\tat wiremock.jakarta.servlet.http.HttpServlet.service(HttpServlet.java:587)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:764)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1665)\n\tat wiremock.org.eclipse.jetty.servlets.CrossOriginFilter.handle(CrossOriginFilter.java:314)\n\tat wiremock.org.eclipse.jetty.servlets.CrossOriginFilter.doFilter(CrossOriginFilter.java:267)\n\tat wiremock.org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:202)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1635)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:527)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:221)\n\tat wiremock.org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1381)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:176)\n\tat wiremock.org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:484)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:174)\n\tat wiremock.org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1303)\n\tat wiremock.org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:129)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)\n\tat wiremock.org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:822)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:141)\n\tat wiremock.org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)\n\tat wiremock.org.eclipse.jetty.server.Server.handle(Server.java:563)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel$RequestDispatchable.dispatch(HttpChannel.java:1598)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:753)\n\tat wiremock.org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:501)\n\tat wiremock.org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:287)\n\tat wiremock.org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314)\n\tat wiremock.org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100)\n\tat wiremock.org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.runTask(AdaptiveExecutionStrategy.java:421)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.consumeTask(AdaptiveExecutionStrategy.java:390)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.tryProduce(AdaptiveExecutionStrategy.java:277)\n\tat wiremock.org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.run(AdaptiveExecutionStrategy.java:199)\n\tat wiremock.org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:411)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:969)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1194)\n\tat wiremock.org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1149)\n\tat java.base/java.lang.Thread.run(Unknown Source)\n</pre>\n\n</body>\n</html>\n, cause=org.springframework.web.reactive.function.client.WebClientResponseException$InternalServerError: 500 Internal Server Error from POST https://develop-sirius-mocks.sncf-dsi-pr-dev.aws.vsct.fr/gorail)\r\n\tat sncf.traction.sirius.webclientutils.WebClientError$Companion.from(WebClientExt.kt:55)\r\n\tat sncf.traction.sirius.partners.gorail.GorailClient.searchSillonsDasSfera-T3BZVFY$suspendImpl(GorailClient.kt:260)\r\n\tat sncf.traction.sirius.partners.gorail.GorailClient$searchSillonsDasSfera$1.invokeSuspend(GorailClient.kt)\r\n\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\r\n\tat kotlinx.coroutines.DispatchedTaskKt.resume(DispatchedTask.kt:233)\r\n\tat kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:175)\r\n\tat kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:147)\r\n\tat kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)\r\n\tat kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)\r\n\tat kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)\r\n\tat kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)\r\n\tat kotlinx.coroutines.reactor.MonoKt$awaitSingleOrNull$2$1.onError(Mono.kt:62)\r\n\tat reactor.core.publisher.StrictSubscriber.onError(StrictSubscriber.java:106)\r\n\tat reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onError(FluxContextWrite.java:121)\r\n\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241)\r\n\tat reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315)\r\n\tat reactor.core.publisher.Operators$MultiSubscriptionSubscriber.onError(Operators.java:2236)\r\n\tat reactor.core.publisher.FluxOnAssembly$OnAssemblySubscriber.onError(FluxOnAssembly.java:544)\r\n\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180)\r\n\tat reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onError(MonoIgnoreThen.java:280)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:232)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onComplete(MonoIgnoreThen.java:204)\r\n\tat reactor.core.publisher.FluxOnErrorReturn$ReturnSubscriber.onComplete(FluxOnErrorReturn.java:169)\r\n\tat reactor.core.publisher.MonoIgnoreElements$IgnoreElementsSubscriber.onComplete(MonoIgnoreElements.java:89)\r\n\tat reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onComplete(FluxMapFuseable.java:152)\r\n\tat reactor.core.publisher.FluxOnAssembly$OnAssemblySubscriber.onComplete(FluxOnAssembly.java:549)\r\n\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\tat reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)\r\n\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\tat reactor.core.publisher.Operators.complete(Operators.java:137)\r\n\tat reactor.netty.channel.FluxReceive.startReceiver(FluxReceive.java:179)\r\n\tat reactor.netty.channel.FluxReceive.subscribe(FluxReceive.java:145)\r\n\tat reactor.core.publisher.InternalFluxOperator.subscribe(InternalFluxOperator.java:68)\r\n\tat reactor.netty.ByteBufFlux.subscribe(ByteBufFlux.java:340)\r\n\tat reactor.core.publisher.Mono.subscribe(Mono.java:4576)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:265)\r\n\tat reactor.core.publisher.MonoIgnoreThen.subscribe(MonoIgnoreThen.java:51)\r\n\tat reactor.core.publisher.Mono.subscribe(Mono.java:4576)\r\n\tat reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)\r\n\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241)\r\n\tat reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onError(MonoIgnoreThen.java:280)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:232)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.onComplete(MonoIgnoreThen.java:204)\r\n\tat reactor.core.publisher.FluxOnErrorReturn$ReturnSubscriber.onComplete(FluxOnErrorReturn.java:169)\r\n\tat reactor.core.publisher.MonoIgnoreElements$IgnoreElementsSubscriber.onComplete(MonoIgnoreElements.java:89)\r\n\tat reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onComplete(FluxMapFuseable.java:152)\r\n\tat reactor.core.publisher.FluxOnAssembly$OnAssemblySubscriber.onComplete(FluxOnAssembly.java:549)\r\n\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\tat reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)\r\n\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\tat reactor.core.publisher.Operators.complete(Operators.java:137)\r\n\tat reactor.netty.channel.FluxReceive.startReceiver(FluxReceive.java:179)\r\n\tat reactor.netty.channel.FluxReceive.subscribe(FluxReceive.java:145)\r\n\tat reactor.core.publisher.InternalFluxOperator.subscribe(InternalFluxOperator.java:68)\r\n\tat reactor.netty.ByteBufFlux.subscribe(ByteBufFlux.java:340)\r\n\tat reactor.core.publisher.Mono.subscribe(Mono.java:4576)\r\n\tat reactor.core.publisher.MonoIgnoreThen$ThenIgnoreMain.subscribeNext(MonoIgnoreThen.java:265)\r\n\tat reactor.core.publisher.MonoIgnoreThen.subscribe(MonoIgnoreThen.java:51)\r\n\tat reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:165)\r\n\tat reactor.core.publisher.FluxMap$MapSubscriber.onNext(FluxMap.java:122)\r\n\tat reactor.core.publisher.FluxOnErrorReturn$ReturnSubscriber.onNext(FluxOnErrorReturn.java:162)\r\n\tat reactor.core.publisher.FluxDefaultIfEmpty$DefaultIfEmptySubscriber.onNext(FluxDefaultIfEmpty.java:122)\r\n\tat reactor.core.publisher.FluxOnAssembly$OnAssemblySubscriber.onNext(FluxOnAssembly.java:539)\r\n\tat reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onNext(FluxMapFuseable.java:129)\r\n\tat reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onNext(FluxContextWrite.java:107)\r\n\tat reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.onNext(FluxMapFuseable.java:299)\r\n\tat reactor.core.publisher.FluxFilterFuseable$FilterFuseableConditionalSubscriber.onNext(FluxFilterFuseable.java:337)\r\n\tat reactor.core.publisher.Operators$BaseFluxToMonoOperator.completePossiblyEmpty(Operators.java:2097)\r\n\tat reactor.core.publisher.MonoCollect$CollectSubscriber.onComplete(MonoCollect.java:145)\r\n\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\tat reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)\r\n\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\tat reactor.netty.channel.FluxReceive.onInboundComplete(FluxReceive.java:413)\r\n\tat reactor.netty.channel.ChannelOperations.onInboundComplete(ChannelOperations.java:455)\r\n\tat reactor.netty.channel.ChannelOperations.terminate(ChannelOperations.java:509)\r\n\tat reactor.netty.http.client.HttpClientOperations.onInboundNext(HttpClientOperations.java:819)\r\n\tat reactor.netty.channel.ChannelOperationsHandler.channelRead(ChannelOperationsHandler.java:115)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\tat org.zalando.logbook.netty.LogbookClientHandler.channelRead(LogbookClientHandler.java:75)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\tat io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:289)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\tat io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)\r\n\tat io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)\r\n\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)\r\n\tat io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\tat io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1515)\r\n\tat io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1378)\r\n\tat io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1427)\r\n\tat io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:530)\r\n\tat io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:469)\r\n\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\tat io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1357)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)\r\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\tat io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:868)\r\n\tat io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)\r\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)\r\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)\r\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)\r\n\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)\r\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)\r\n\tat io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\r\n\tat io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\nCaused by: org.springframework.web.reactive.function.client.WebClientResponseException$InternalServerError: 500 Internal Server Error from POST https://develop-sirius-mocks.sncf-dsi-pr-dev.aws.vsct.fr/gorail\r\n\tat org.springframework.web.reactive.function.client.WebClientResponseException.create(WebClientResponseException.java:332)\r\n\tSuppressed: The stacktrace has been enhanced by Reactor, refer to additional information below: \r\nError has been observed at the following site(s):\r\n\t*__checkpoint ⇢ 500 INTERNAL_SERVER_ERROR from POST https://develop-sirius-mocks.sncf-dsi-pr-dev.aws.vsct.fr/gorail [DefaultWebClient]\r\nOriginal Stack Trace:\r\n\t\tat org.springframework.web.reactive.function.client.WebClientResponseException.create(WebClientResponseException.java:332)\r\n\t\tat org.springframework.web.reactive.function.client.DefaultClientResponse.lambda$createException$1(DefaultClientResponse.java:214)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onNext(FluxMap.java:106)\r\n\t\tat reactor.core.publisher.FluxOnErrorReturn$ReturnSubscriber.onNext(FluxOnErrorReturn.java:162)\r\n\t\tat reactor.core.publisher.FluxDefaultIfEmpty$DefaultIfEmptySubscriber.onNext(FluxDefaultIfEmpty.java:122)\r\n\t\tat reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onNext(FluxMapFuseable.java:129)\r\n\t\tat reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onNext(FluxContextWrite.java:107)\r\n\t\tat reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.onNext(FluxMapFuseable.java:299)\r\n\t\tat reactor.core.publisher.FluxFilterFuseable$FilterFuseableConditionalSubscriber.onNext(FluxFilterFuseable.java:337)\r\n\t\tat reactor.core.publisher.Operators$BaseFluxToMonoOperator.completePossiblyEmpty(Operators.java:2097)\r\n\t\tat reactor.core.publisher.MonoCollect$CollectSubscriber.onComplete(MonoCollect.java:145)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)\r\n\t\tat reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)\r\n\t\tat reactor.netty.channel.FluxReceive.onInboundComplete(FluxReceive.java:413)\r\n\t\tat reactor.netty.channel.ChannelOperations.onInboundComplete(ChannelOperations.java:455)\r\n\t\tat reactor.netty.channel.ChannelOperations.terminate(ChannelOperations.java:509)\r\n\t\tat reactor.netty.http.client.HttpClientOperations.onInboundNext(HttpClientOperations.java:819)\r\n\t\tat reactor.netty.channel.ChannelOperationsHandler.channelRead(ChannelOperationsHandler.java:115)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat org.zalando.logbook.netty.LogbookClientHandler.channelRead(LogbookClientHandler.java:75)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:289)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)\r\n\t\tat io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1515)\r\n\t\tat io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1378)\r\n\t\tat io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1427)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:530)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:469)\r\n\t\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\r\n\t\tat io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1357)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)\r\n\t\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\r\n\t\tat io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:868)\r\n\t\tat io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)\r\n\t\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)\r\n\t\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)\r\n\t\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)\r\n\t\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)\r\n\t\tat io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)\r\n\t\tat io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\r\n\t\tat io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)\r\n\t\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\n"}}}
```

--- a/logbook-core/src/main/java/org/zalando/logbook/core/FilteredHttpRequest.java
+++ b/logbook-core/src/main/java/org/zalando/logbook/core/FilteredHttpRequest.java
@@ -1,5 +1,6 @@
 package org.zalando.logbook.core;
 
+import java.io.IOException;
 import lombok.AllArgsConstructor;
 import org.apiguardian.api.API;
 import org.zalando.logbook.BodyFilter;
@@ -10,8 +11,7 @@ import org.zalando.logbook.HttpRequest;
 import org.zalando.logbook.PathFilter;
 import org.zalando.logbook.QueryFilter;
 import org.zalando.logbook.RequestURI;
-
-import java.io.IOException;
+import org.zalando.logbook.attributes.HttpAttributes;
 
 import static lombok.AccessLevel.PRIVATE;
 import static org.apiguardian.api.API.Status.INTERNAL;
@@ -27,12 +27,13 @@ final class FilteredHttpRequest implements ForwardingHttpRequest {
     private final BodyFilter bodyFilter;
 
     private final HttpHeaders headers;
+    private final HttpAttributes attributes;
 
     FilteredHttpRequest(final HttpRequest request,
-            final QueryFilter queryFilter,
-            final PathFilter pathFilter,
-            final HeaderFilter headerFilter,
-            final BodyFilter bodyFilter) {
+                        final QueryFilter queryFilter,
+                        final PathFilter pathFilter,
+                        final HeaderFilter headerFilter,
+                        final BodyFilter bodyFilter) {
         this.request = request;
         this.bodyFilter = bodyFilter;
         this.headers = headerFilter.filter(request.getHeaders());
@@ -41,6 +42,7 @@ final class FilteredHttpRequest implements ForwardingHttpRequest {
         this.query = query.isEmpty() ? query : queryFilter.filter(query);
 
         this.path = pathFilter.filter(request.getPath());
+        this.attributes = request.getAttributes();
     }
 
     @Override
@@ -69,17 +71,22 @@ final class FilteredHttpRequest implements ForwardingHttpRequest {
     }
 
     @Override
+    public HttpAttributes getAttributes() {
+        return attributes;
+    }
+
+    @Override
     public HttpRequest withBody() throws IOException {
-        return withRequest(request.withBody());
+        return withRequest(request.withBody(), request.getAttributes());
     }
 
     @Override
     public HttpRequest withoutBody() {
-        return withRequest(request.withoutBody());
+        return withRequest(request.withoutBody(), request.getAttributes());
     }
 
-    private FilteredHttpRequest withRequest(final HttpRequest request) {
-        return new FilteredHttpRequest(request, query, path, bodyFilter, headers);
+    private FilteredHttpRequest withRequest(final HttpRequest request, final HttpAttributes attributes) {
+        return new FilteredHttpRequest(request, query, path, bodyFilter, headers, attributes);
     }
 
     @Override

--- a/logbook-core/src/test/java/org/zalando/logbook/core/FilteredHttpRequestTest.java
+++ b/logbook-core/src/test/java/org/zalando/logbook/core/FilteredHttpRequestTest.java
@@ -1,20 +1,31 @@
 package org.zalando.logbook.core;
 
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
 import org.junit.jupiter.api.Test;
 import org.zalando.logbook.BodyFilter;
 import org.zalando.logbook.HeaderFilter;
 import org.zalando.logbook.HttpHeaders;
 import org.zalando.logbook.HttpRequest;
 import org.zalando.logbook.PathFilter;
+import org.zalando.logbook.attributes.HttpAttributes;
 import org.zalando.logbook.test.MockHttpRequest;
-
-import java.io.IOException;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 final class FilteredHttpRequestTest {
+    private final Map<String, Object> attributesMap = new HashMap<>();
+
+    {
+        attributesMap.put("subject", "John");
+        attributesMap.put("object", "Window");
+    }
+
+    private final HttpAttributes httpAttributes = new HttpAttributes(attributesMap);
 
     private final HttpRequest unit = buildFilteredHttpRequest(PathFilters.replace("/endpoint/{secrets}/action", "XXX"));
+    private final HttpRequest unitWithHttpAttributes = buildFilteredHttpRequest(PathFilters.replace("/endpoint/{secrets}/action", "XXX"), httpAttributes);
     private final HttpRequest unitWithDynamicPathFilterReplacement = buildFilteredHttpRequest(PathFilters.replace("/endpoint/{secrets}/action", String::toUpperCase));
 
     @Test
@@ -101,7 +112,34 @@ final class FilteredHttpRequestTest {
         assertThat(unitWithDynamicPathFilterReplacement.getPath()).isEqualTo("/endpoint/SECRET/action");
     }
 
+    @Test
+    void shouldKeepHttpAttributes() throws IOException {
+        assertThat(unitWithHttpAttributes.getAttributes()).isEqualTo(httpAttributes);
+        assertThat(unit.getAttributes()).isEqualTo(HttpAttributes.EMPTY);
+        assertThat(unitWithHttpAttributes.withoutBody().getAttributes()).isEqualTo(httpAttributes);
+        assertThat(unitWithHttpAttributes.withBody().getAttributes()).isEqualTo(httpAttributes);
+        assertThat(unit.withoutBody().getAttributes()).isEqualTo(HttpAttributes.EMPTY);
+        assertThat(unit.withBody().getAttributes()).isEqualTo(HttpAttributes.EMPTY);
+    }
+
+    private FilteredHttpRequest buildFilteredHttpRequest(PathFilter pathFilter, HttpAttributes httpAttributes) {
+
+        return new FilteredHttpRequest(MockHttpRequest.create()
+                .withQuery("password=1234&limit=1")
+                .withHeaders(HttpHeaders.empty()
+                        .update("Authorization", "Bearer 9b7606a6-6838-11e5-8ed4-10ddb1ee7671")
+                        .update("Accept", "text/plain"))
+                .withBodyAsString("My secret is s3cr3t")
+                .withPath("/endpoint/secret/action")
+                .withHttpAttributes(httpAttributes),
+                QueryFilters.replaceQuery("password", "unknown"),
+                pathFilter,
+                HeaderFilters.authorization(),
+                (contentType, body) -> body.replace("s3cr3t", "f4k3"));
+    }
+
     private FilteredHttpRequest buildFilteredHttpRequest(PathFilter pathFilter) {
+
         return new FilteredHttpRequest(MockHttpRequest.create()
                 .withQuery("password=1234&limit=1")
                 .withHeaders(HttpHeaders.empty()

